modules/sbo-release-notes-1-3.adoc

```diff
@@ -0,0 +1,12 @@
+[id="sbo-release-notes-1-3_{context}"]
+// Module included in the following assembly:
+//
+// * applications/connecting_applications_to_services/sbo-release-notes.adoc
+:_content-type: REFERENCE
```

Comment:
The `content-type` metadata needs to be placed before the anchor ID:

```
// Module included in the following assembly:
//
// * applications/connecting_applications_to_services/sbo-release-notes.adoc

:_content-type: REFERENCE
[id="sbo-release-notes-1-3_{context}"]
= Release notes for {servicebinding-title} 1.3
```

Reply:
done